apps/web/app/layouts/footer/footer.tsx

```diff
@@ -49,7 +49,7 @@ function AppFooter() {
         product="A Trust Machines product"
         // Hard coding date as Cloudflare Workers renders new Date() as epoch
         // start time
-        copyright={`© 2025 Leather Wallet, LLC`}
+        copyright="© 2025 Leather Wallet, LLC"
```

Comment:
Should be 2026 right?

Reply:
Not sure if a joke about release date lol

Reply:
yes it is, some Friday fun 😎